src/app/(auth)/login/page.tsx

```diff
@@ -1,5 +1,118 @@
-import type { FC } from "react";
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+"use client";
 
-export const LoginPage: FC = () => {
-  return <div>LoginPage</div>;
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import * as z from "zod";
+import Image from "next/image";
+import { Mail, Lock } from "lucide-react";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import tabletImage from "@/public/svg/fragments/tablet-login.svg";
+import BgAuth from "@/components/home/backgrounds/bg-auth";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+
+const LoginPage = () => {
+  const formSchema = z.object({
+    email: z.string().email({ message: "Invalid email address" }),
+    password: z
+      .string()
+      .min(6, { message: "Password must be at least 6 characters" })
+      .regex(/[A-Z]/, {
+        message: "Password must contain at least 1 capital letter",
+      }),
+  });
+  const form = useForm<z.infer<typeof formSchema>>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      email: "",
+      password: "",
+    },
+  });
+  function onSubmit(values: z.infer<typeof formSchema>) {
+    alert(values);
+  }
+
+  return (
+    <main className="flex h-screen  justify-center relative">
+      <BgAuth />
+      <section className="w-full mx-0 md:mx-8 h-screen md:h-5/6 sm:overflow-hidden rounded-none md:rounded-3xl bg-white shadow-2xl self-center">
```

Comment:
For mobile screen, just hide the illustration image. Use `h-fit` and `mx-5` and maintain the border radius for the container